script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -104,18 +104,18 @@ I've gotten so used to an ordinary house that living at a place like the Tohno m
 ".......To begin with,
 I can't imagine a Western-style building bigger than my school......"
 I'm amazed at the things I was able to endure as a child.
-The mere thought of being returned to such an anachronistic lifestyle makes me feel depressed, even if I don't feel anxious.
+The mere thought of returning to such an anachronistic lifestyle makes me feel depressed, even if I don't feel anxious.
 <Page29>
 The train arrives at a large station and stops for several minutes.
-Since this train stops at every station, I had to wait for the next rapid train to pass by.
+Since this train is stopping at every station, I have waited for the next rapid train to pass by.
```

Comment:
Hmm, I think the previous fits better here when it comes to "had to wait" / "have waited" - it puts the inconvenience of having to wait more forward, and "have waited" has more of a longer timescale / ongoing event feeling. Hard to describe exactly why it reads strangely, sorry.